tests/core/test_policies.py

```diff
@@ -1046,9 +400,13 @@ def test_get_training_trackers_for_policy(
     assert len(ml_trackers) == n_ml_trackers
 
 
-@pytest.mark.parametrize(
-    "policy", [FormPolicy, MappingPolicy, FallbackPolicy, TwoStageFallbackPolicy]
-)
-def test_deprecation_warnings_for_old_rule_like_policies(policy: Type[Policy]):
-    with pytest.warns(FutureWarning):
-        policy(None)
+class PolicyWithoutInitKwargs(Policy):
+    def __init__(self, *args: Any) -> None:
+        pass
+
+    def persist(self, _) -> None:
+        pass
+
+    @classmethod
+    def _metadata_filename(cls) -> Text:
+        return "no_finetune_policy"
```

Comment:
Looks like this policy isn't used anywhere, did you mean to add some more code?

Reply:
Ahh that should be removed. Its leftover code that should have been deleted when I was resolving a merge conflict with main. Removing it now.